lib/active_merchant/billing/gateways/mit.rb

```diff
@@ -0,0 +1,244 @@
+        engine.iv = iv
+        # Encrypts the texts and stores the bytes
+        encrypted_bytes = engine.update(val) + engine.final
+        # Concatenates the (a) IV bytes and (b) the encrypted bytes then returns a
+        # base64 representation
+        [iv << encrypted_bytes].pack('m')
+      end
+
+      def authorize(money, payment, options = {})
+        puts('---------------------------------------------')
+        puts('Authorization')
+        puts('---------------------------------------------')
+        post = {
+          operation: 'Authorize',
+          id_comercio: self.options[:id_comercio],
+          user: self.options[:user],
+          apikey: self.options[:apikey]
+        }
+        add_invoice(post, money, options)
+        # Payments contains the card information
+        add_payment(post, payment)
+        # Address not required
+        add_address(post, payment, options)
+        add_customer_data(post, options)
+        post[:key_session] = self.options[:key_session]
+
+        post_to_json = post.to_json
+        puts(post_to_json)
+        post_to_json_encrypt = encrypt(post_to_json, self.options[:key_session])
+
+        final_post = post_to_json_encrypt + '-' + self.options[:user]
+        commit('sale', final_post)
+      end
+
+      def capture(money, authorization, options = {})
+        puts('---------------------------------------------')
+        puts('Capture')
+        puts('---------------------------------------------')
+        post = {
+          operation: 'Capture',
+          id_comercio: self.options[:id_comercio],
+          user: self.options[:user],
+          apikey: self.options[:apikey],
+          transaccion_id: options[:transaccion_id],
+          amount: amount(money)
+        }
+        post[:key_session] = self.options[:key_session]
+
+        post_to_json = post.to_json
+        puts(post_to_json)
+        post_to_json_encrypt = encrypt(post_to_json, self.options[:key_session])
+
+        final_post = post_to_json_encrypt + '-' + self.options[:user]
+        commit('capture', final_post)
+      end
+
+      def refund(money, authorization, options = {})
+        puts('---------------------------------------------')
+        puts('Refund')
+        puts('---------------------------------------------')
+        post = {
+          operation: 'Refund',
+          id_comercio: self.options[:id_comercio],
+          user: self.options[:user],
+          apikey: self.options[:apikey],
+          transaccion_id: options[:transaccion_id],
+          auth: authorization,
+          amount: amount(money)
+        }
+        post[:key_session] = self.options[:key_session]
+
+        post_to_json = post.to_json
+        puts(post_to_json)
+        post_to_json_encrypt = encrypt(post_to_json, self.options[:key_session])
+
+        final_post = post_to_json_encrypt + '-' + self.options[:user]
+        commit('refund', final_post)
+      end
+
+      # Currently unsupported
```

Comment:
Since this is unsupported, I'd recommend that it be removed instead of simply being commented out.